test/e2e/atlas/deployments_local_test.go

```diff
@@ -186,33 +184,35 @@ test   LOCAL   7.0.1     IDLE
 		out := string(r)
 		req.NoError(err, out)
 		a := assert.New(t)
-		a.Contains(out, createMessage)
+		a.Contains(out, "Search index created")
 	})
 
 	var indexID string
-	t.Run("Wait for search index", func(t *testing.T) {
-		for {
-			t.Log("Waiting for index...")
-			cursor, err := myCol.Aggregate(ctx, mongo.Pipeline{
-				{
-					{Key: "$listSearchIndexes", Value: bson.D{}},
-				},
-			})
-			req.NoError(err)
-			var results []bson.M
-			req.NoError(cursor.All(ctx, &results))
-			if len(results) == 0 {
-				continue // no index found
-			}
-			status, ok := results[0]["status"].(string)
-			if !ok {
-				continue // no status found
-			}
-			if status == "STEADY" {
-				indexID, _ = results[0]["id"].(string)
-				break
-			}
-		}
+
+	t.Run("Index List", func(t *testing.T) {
+		cmd := exec.Command(cliPath,
+			deploymentEntity,
+			searchEntity,
```

Comment:
Why do we need this? we are testing index list at line 254. Can we remove it?

Reply:
I had to move test sooner, deleted the duplication, it was a bad merge